source/clear-linux/reference/faqs.rst

```diff
@@ -0,0 +1,200 @@
+.. _faqs:
+
+Community FAQs
+###############
+
+Below is a list of commonly asked questions with answers sourced from the
```

Comment:
For community sourced questions - do we want to link back to the originating thread in Discourse?

Reply:
I would say - Yes, if they provide answers/solutions. No, if it is just to link to the origin of the question (which can be multiple places, hence FAQ).

Reply:
Definitely need a solid answer to a FAQ.

Reply:
Agreed.